### PR TITLE
Add run shortcut

### PR DESCRIPTION
**Pull request by liberaldev**
Add a keyboard shortcut to run code without having to leave the keyboard. The shortcut is Crtl+Enter.

### CODE REVIEW

File: app/try_ruby.rb [hunk 1]
```diff
@@ -59,6 +59,24 @@ def initialize
     # Bind run button
     $document.on(:click, '#btn_run') { do_run }
 
+    # Control key pressed or not
+    control_pressed = false
+
+    #If you hold down the control and the Enter key goes down, run
+    $document.on :keydown, '#editor' do |e|
```
Comment (hmdne) [inline app/try_ruby.rb:63]: Perhaps `keypress`?

File: app/try_ruby.rb [hunk 2]
```diff
@@ -59,6 +59,24 @@ def initialize
...
+      if e.key == "Control"
+        control_pressed = true
+      elsif e.key == "Enter" && control_pressed
```
Comment (hmdne) [inline app/try_ruby.rb]: Perhaps `e.ctrl?`? If both changes are applied, the code could be much simpler :)

**Comment by hmdne** [comment]
@liberaldev Thanks! Can you try the mentioned changes? Perhaps the code can be simplified this way a little.

**Comment by liberaldev** [comment]
> @liberaldev Thanks! Can you try the mentioned changes? Perhaps the code can be simplified this way a little.

Neither works as intended.

File: app/try_ruby.rb (hunk 1 above)
Comment (liberaldev) [reply app/try_ruby.rb:63]: If you make the changes as suggested, your code will not behave as intended.

File: app/try_ruby.rb (hunk 2 above)
Comment (liberaldev) [reply app/try_ruby.rb:69]: Also, if you change the code as suggested, it will not behave as intended.

**Comment by liberaldev** [comment]
> @liberaldev Thanks! Can you try the mentioned changes? Perhaps the code can be simplified this way a little.

I simplified the code.

File: app/try_ruby.rb (hunk 2 above)
Comment (liberaldev) [reply app/try_ruby.rb]: I've simplified the code to reflect this.

**Pull request merged**